### PR TITLE
Is trustred#27

### PR DESCRIPTION
**Pull request by sculove**
### Details
- add dependency libs
- add isTrusted property
- add release/changelog tasks

### Issues
#6

### CODE REVIEW

File: src/Visible.js [hunk 1]
```diff
@@ -244,13 +244,17 @@ class Visible extends Component {
...
+		 * @event
+		 * @type {obejct} The object of data to be sent when the event is fired<ko>이벤트가 발생할 때 전달되는 데이터 객체</ko>
+		 * @property {Array} visible Visible elements  (the element type is `HTMLElement`) <ko>보이게 된 엘리먼트들</ko>
+		 * @property {Array} invisible Invisible elements  (the element type is `HTMLElement`) <ko>안 보이게 된 엘리먼트들</ko>
+		 * @property {Boolean} isTrusted Returns true if an event was generated by the user action, or false if it was caused by a script or API call <ko>사용자의 액션에 의해 이벤트가 발생하였으면 true, 스크립트나 API호출에 의해 발생하였을 경우에는 false를 반환한다.</ko>
 		 */
 		this.trigger("change", {
 			visible: visibles,
 			invisible: invisibles,
+			isTrusted: true, // eg.Visible event is called by 'check' method.
```
Comment (jongmoon) [inline src/Visible.js]: @sculove Is `isTrusted` needed although `isTrusted` is always `true`?

Comment (sculove) [reply src/Visible.js]: Yes. `isTrusted` is coding convention of egjs.

Comment (jongmoon) [reply src/Visible.js]: @sculove Thank you. 

By the way, comment `eg.Visible event is called by 'check' method.`. need to be fixed like follows `This event is called by 'check' method.`

Comment (sculove) [reply src/Visible.js]: @jongmoon ok. thanks

File: src/utils.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	return array;
+}
+
+/**
+ * Select or create element
+ * @param {String|HTMLElement|jQuery} param
+ *  when string given is as HTML tag, then create element
+ *  otherwise it returns selected elements
+ * @param {Boolean} multi
+ * @returns {HTMLElement}
```
Comment (jongmoon) [inline src/utils.js]: @sculove Need to say it returns Array of HTMLElement

Comment (sculove) [reply src/utils.js]: ok thanks.

**Comment by jongmoon** [comment]
LGTM

**Pull request merged**